llmware/library.py

```diff
@@ -746,14 +1181,42 @@ def add_office(self, input_folder=None):
 
     def get_all_library_cards(self, account_name='llmware'):
 
-        """ Get all library cards for all libraries on account """
+        """ Get all library cards for all libraries on account 
+        
+            Parameters
+            ----------
+            account_name : str, default='llmware'
+                The name of the account for which to retrieve all library cards.
+
+            Returns
+            -------
+            library_cards : list of dict
+                A list of all library card dictionaries for the specified account.
+        """
 
         library_cards = LibraryCatalog(account_name=account_name).all_library_cards()
         return library_cards
 
     def delete_installed_embedding(self, embedding_model_name, vector_db, vector_db_api_key=None):
 
-        """ Deletes an installed embedding on specific combination of vector_db + embedding_model_name """
+        """ Deletes an installed embedding on specific combination of vector_db + embedding_model_name 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Deletes an installed embedding on specific combination of vector_db + embedding_model_name.
```